tests/integration/registration.controller.int.test.js

```diff
@@ -0,0 +1,12 @@
+const request = require('supertest')
+const app = require('../../app')
+const newUser = require('../mock-data/new-user.json')
+
+const registrationUrl = '/api/accounts/new/'
+
+describe(registrationUrl, () => {
+  it('should return 500 on POST for already existing users' + registrationUrl, async () => {
+    const res = await request(app).post(registrationUrl).send(newUser)
+    expect(res.statusCode).toBe(500)
```

Comment:
500 series of status codes are for when an application cannot process the request for reasons like they cannot connect to the db. I think a 422 or something in the 400 series would be more correct.